script.js

```diff
@@ -1,12 +1,33 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let angle = 0;
+let color1;
+let color2;
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+  background(0);
+  frameRate(40);
+  rectMode(CENTER);
+  angleMode(DEGREES);
+  color1 = color(255, 0, 0, 70);
```

Comment:
I like to comment what color something is when hard-code a color. or even name the variable that color. just nice for someone reading the code so they don't have to figure out what the color is.

Reply:
the opacity was a great move! really adds to the aesthetic

Reply:
Ohh, thank you for letting me know, that makes a lot of sense for readability.